### PR TITLE
[Tabs] Add bottom divider.

### LINKED ISSUE #7742: [Tabs] Internal issue: b/136250383
This was filed as an internal issue. If you are a Googler, please visit [b/136250383](http://b/136250383) for more details.

---
#### Internal data
- Associated internal bug: [b/136250383](http://b/136250383)

### PR DESCRIPTION
**Pull request by romoore**
Adds the bottom divider and an API to make it visible and customize its color.

## TabBarView Example View Controller
|Before|After|
|---|---|
|Simulator Screen Shot - iPhone 7 - 2019-07-08 at 23 39 56|Simulator Screen Shot - iPhone 7 - 2019-07-08 at 23 34 30|

Closes #7742

### CODE REVIEW

File: components/Tabs/src/TabBarView/MDCTabBarView.h [hunk 1]
```diff
@@ -34,6 +34,9 @@ __attribute__((objc_subclassing_restricted)) @interface MDCTabBarView : UIScroll
 /** The color of the Tab bar's background. */
 @property(nullable, nonatomic, copy) UIColor *barTintColor;
 
+/** The color of the bottom divider. Default is clear. */
+@property(nonnull, nonatomic, copy) UIColor *bottomDividerColor;
```
Comment (codeman7) [inline components/Tabs/src/TabBarView/MDCTabBarView.h:38]: How do you feel about a `BOOL showsDivider` as we have in [Banner](https://github.com/material-components/material-components-ios/blob/4ed8860b20d76dbadf527225e2dbf037feaa4029/components/Banner/src/MDCBannerView.h#L78)

File: components/Tabs/src/TabBarView/MDCTabBarView.m [hunk 2]
```diff
@@ -493,6 +510,8 @@ - (void)layoutSubviews {
   } else {
     [self layoutSubviewsForScrollableLayout];
   }
+  self.bottomDividerView.frame = CGRectMake(
+      CGRectGetMinX(self.bounds), CGRectGetMaxY(self.bounds) - 1, CGRectGetWidth(self.bounds), 1);
```
Comment (codeman7) [inline components/Tabs/src/TabBarView/MDCTabBarView.m]: Should these be constants?

**Review by codeman7 — APPROVED** [review]

File: components/Tabs/src/TabBarView/MDCTabBarView.h (hunk 1 above)
Comment (romoore) [reply components/Tabs/src/TabBarView/MDCTabBarView.h:38]: If the divider affected the layout/sizing, I would agree. On Tabs, though, the divider does not have any impact other than obscuring a 1-point-high portion on the bottom.  I'd like to keep the current API for now and we can add another if clients find it confusing later.

File: components/Tabs/src/TabBarView/MDCTabBarView.m (hunk 2 above)
Comment (romoore) [reply components/Tabs/src/TabBarView/MDCTabBarView.m]: Good idea. Done.

**Pull request merged**